srcpkgs/nvimpager/template

```diff
@@ -0,0 +1,36 @@
+# Template file for 'nvimpager'
+pkgname=nvimpager
+version=0.6
+revision=1
+create_wrksrc=true
+build_wrksrc="${pkgname}-${version}"
+build_style=gnu-makefile
+make_build_target="nvimpager.configured"
+make_build_args="PREFIX=/usr"
+hostmakedepends="neovim pandoc"
+depends="neovim>=0.3.0 bash"
+short_desc="Wrapper to use neovim as a pager to view manpages, diffs, etc"
+maintainer="Adrian Herath <adrianisuru@gmail.com>"
+license="BSD-2-Clause"
+homepage="https://github.com/lucc/nvimpager"
+distfiles="${homepage}/archive/v${version}.tar.gz
+ https://www.vim.org/scripts/download_script.php?src_id=14498>AnsiEsc.vba.gz"
+checksum="0415a57ae53d20a9ecdc5d85c22a2847319631f31d9c5b09fe22506183148c46
+ a80d8c95a2e497e190831f1b580205653c595c6f51b48d4cb89fb33eaea5ed03"
+
+pre_build() {
+# Date is the date of the latest commit in the nvimpager repo.
+# It should be changed when updating to a new version.
+	cat <<< \
+"---
+footer: Version v${version}
+date: 2019-03-05
```

Comment:
date still not used somehow?

Reply:
I don't understand what you mean by this. What should I change here?

Reply:
I do not see this date being used anywhere in pager. Unlikely a problem, just letting you know, because you try to make it effective to something.

Reply:
Got it.